File: 2020/2020W38/PreppindataW38.py
import pandas as pd

final = pd.read_excel(pd.ExcelFile("F:/Data/Input - Anagrams.xlsx"),'Anagrams')  
final['Anagram?'] = final.apply(lambda x: sorted(list(x['Word 1'].lower()))==sorted(list(x['Word 2'].lower())), axis=1)

File: 2020/2020W38/preppindataw38.R
library(dplyr)
library(readxl)

final <- read_excel("F:/Data/Input - Anagrams.xlsx", sheet = "Anagrams") %>%
  rowwise() %>%
  mutate('Anagram?'= (paste(sort(unlist(strsplit(tolower(`Word 1`), ''))), collapse = '')==
                        paste(sort(unlist(strsplit(tolower(`Word 2`), ''))), collapse = '')))

View(final)
